content/application-service-api.md

```diff
@@ -436,6 +436,10 @@ an application service-defined namespace will receive the same
 `M_EXCLUSIVE` error code, but only if the application service has
 defined the namespace as `exclusive`.
 
+If either endpoint is called with the `m.login.application_service` login type,
+but without a valid `as_token`, the endpoints will return an error with the
+`M_MISSING_TOKEN` or `M_UNKNOWN_TOKEN` error code.
```

Comment:
should we specify the HTTP error code? 401?

Reply:
Specified 401 and also added a link to the C-S spec which has the same code